src/view/login.js

```diff
@@ -275,18 +276,30 @@ const Login = () => {
               },
             }}
           >
-            <Buttons
-              variant="contained"
-              startIcon={<GoogleIcon />}
-              sx={mediaStyles}
-              value="Login with Google"
-            />
-            <Buttons
-              variant="contained"
-              startIcon={<FacebookRoundedIcon />}
-              sx={mediaStyles}
-              value="Login with Facebook"
-            />
+            <A
+              href={`https://barefoot-backend-development.herokuapp.com/api/v1/users/google/login/?${urlSerializer(
+                `${process.env.DEPLOY_PRIME_URL}/social/login`,
+              )}`}
+            >
+              <Buttons
+                variant="contained"
+                startIcon={<GoogleIcon />}
+                sx={mediaStyles}
+                value="Login with Google"
+              />
+            </A>
+            <A
+              href={`https://barefoot-backend-development.herokuapp.com/api/v1/users/facebook/login/?${urlSerializer(
+                `${process.env.DEPLOY_PRIME_URL}/social/login`,
+              )}`}
+            >
+              <Buttons
+                variant="contained"
+                startIcon={<FacebookRoundedIcon />}
+                sx={mediaStyles}
+                value="Login with Facebook"
+              />
+            </A>
```

Comment:
You must use redux. it's mandatory.

Reply:
implemented it in google_login.js component